Source/ARTRest+Private.h

```diff
@@ -21,6 +21,8 @@ NS_ASSUME_NONNULL_BEGIN
 /// ARTRest private methods that are used internally and for internal testing
 @interface ARTRestInternal : NSObject <ARTRestProtocol, ARTHTTPAuthenticatedExecutor>
 
+typedef void (^CompletionBlock)(NSHTTPURLResponse * _Nullable, NSData * _Nullable, NSError * _Nullable);
+
```

Comment:
Probably something similar was already defined somewhere.

Reply:
can't find it